src/main/java/org/hobbit/core/components/AbstractCommandReceivingComponent.java

```diff
@@ -105,15 +118,22 @@ public void init() throws Exception {
         cmdChannel.exchangeDeclare(Constants.HOBBIT_COMMAND_EXCHANGE_NAME, "fanout", false, true, null);
         cmdChannel.queueBind(queueName, Constants.HOBBIT_COMMAND_EXCHANGE_NAME, "");
 
+        cmdThreadPool = Executors.newCachedThreadPool();
```

Comment:
1. Add two constructors. `AbstractCommandReceivingComponent(boolean execCmdsInParallel)` and `AbstractCommandReceivingComponent() {this(false);}`
2. based on the flag, instantiate cmdThreadPool as cached thread pool or single thread pool.